start-deployBukkitSpigot

```diff
@@ -20,7 +20,16 @@ function buildSpigotFromSource {
     cat /data/spigot_build.log
     exit 1
   fi
-  mv craftbukkit-*.jar /data/${SERVER}
+  if [  $VANILLA_VERSION != "1.15.2" -a \
+        $VANILLA_VERSION != "1.15.1" -a \
+        $VANILLA_VERSION != "1.15" -a \
+        $VANILLA_VERSION != "1.14.4" -a \
+        $VANILLA_VERSION != "1.14.3" -a \
+        $VANILLA_VERSION != "1.14.2" -a \
+        $VANILLA_VERSION != "1.14.1" -a \
+        $VANILLA_VERSION != "1.14" ]; then
```

Comment:
With the new [versionLessThan](https://github.com/itzg/docker-minecraft-server/blob/de866a322ffc55e93c27d055efe4937d58bc19ba/start-utils#L67), I think this could be done with
```suggestion
  if versionLessThan "1.14"; then
```